lib/main.jsx

```diff
@@ -19,6 +19,10 @@ var dummyModel = {
     // This method is for prototype purposes only.
     getContents: function() {
         return TEST_DATA;
+    },
+
+    render: function() {
```

Comment:
It's a hack to force a re-render after moving the IOs in lieu of writing a proper data flow.